priv/templates/phx.gen.auth/registration_live.ex

```diff
@@ -0,0 +1,78 @@
+defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web_namespace, schema.alias) %>RegistrationLive do
+  use <%= inspect context.web_module %>, :live_view
+
+  alias <%= inspect context.module %>
+  alias <%= inspect schema.module %>
+
+  def render(assigns) do
+    ~H"""
+    <h1>Register</h1>
+
+    <.form
+      id="registration_form"
+      let={f}
+      for={@changeset}
+      phx-submit="save"
+      phx-change="validate"
+      phx-trigger-action={@trigger_submit}
+      action={Routes.<%= schema.singular %>_session_path(@socket, :create)}
+      as={:<%= schema.singular %>}
+    >
+      <%%= if @changeset.action do %>
+        <div class="alert alert-danger">
+          <p>Oops, something went wrong! Please check the errors below.</p>
+        </div>
+      <%% end %>
+
+      <%%= label f, :email %>
+      <%%= email_input f, :email, required: true %>
+      <%%= error_tag f, :email %>
+
+      <%%= label f, :password %>
+      <%%= password_input f, :password, required: true %>
```

Comment:
```suggestion
      <%%= password_input f, :password, required: true, value: input_value(f, :password) %>
```
this resolves problem with login after registration, see https://github.com/phoenixframework/phoenix_live_view/issues/70